packages/manager/modules/pci/src/projects/project/serving/onboarding/onboarding.routing.js

```diff
@@ -4,42 +4,26 @@ export default /* @ngInject */ ($stateProvider) => {
     component: 'pciProjectServingOnboarding',
 
     redirectTo: (transition) =>
-      Promise.all([
-        transition.injector().getAsync('lab'),
-        transition.injector().getAsync('namespaces'),
-      ]).then(([lab, namespaces]) => {
-        if (namespaces.length > 0 && !lab.isOpen()) {
-          return { state: 'pci.projects.project.serving' };
-        }
-        return false;
-      }),
+      Promise.all([transition.injector().getAsync('namespaces')]).then(
```

Comment:
You can remove `Promise.all` as you just get one resolve param now.

Reply:
fixed